cognite/neat/graph/loaders/_rdf2dms.py

```diff
@@ -7,8 +8,10 @@
 import yaml
 from cognite.client import CogniteClient
 from cognite.client import data_modeling as dm
+from cognite.client.data_classes.capabilities import Capability, DataModelInstancesAcl
```

Comment:
what is this? :)

Reply:
It checks that the client that is passed have authorization to write to CDF data model instances.